deploy/newrelic-infra-unprivileged.yaml

```diff
@@ -107,6 +107,8 @@ spec:
            #   value: "https://localhost:10257"
            # - name: "API_SERVER_ENDPOINT_URL"
            #   value: "https://localhost:6443"
+           # - name: "NETWORK_ROUTE_FILE"
+           #   value: "/proc/net/route"
```

Comment:
I was trying to make it work in unprivileged by mounting the route file of the host, but every time I mounted the file it had the same information as the container file. I wanted to leave it there in case someone else comes up with something, but I can remove it. WDYT I have no special preference?

Reply:
I don't think this is something that customer will change with frequency, maybe we could leave it in the official docs to declutter the manifests a bit.